ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -394,10 +406,17 @@ def build_peer_mapping(self):
         self.get_portchannel_info()
 
     def build_vlan_if_port_mapping(self):
-        content = self.read_json('vlan_ports_file')
-        if len(content) > 1:
-            raise Exception("Too many vlans")
-        return [(ifname, self.port_indices[ifname]) for ifname in content.values()[0]['members']]
+        portchannel_content = self.read_json('portchannel_ports_file')
+        portchannel_names = [pc['name'] for pc in portchannel_content.values()]
+
+        vlan_content = self.read_json('vlan_ports_file')
+        
+        vlan_if_port = []
+        for vlan in self.vlan_ip_range.keys():
```

Comment:
```suggestion
    for vlan in self.vlan_ip_range:
```
Not a blocking issue, but you can make your code shorter and more pythonic.

Reply:
Thanks for the comment, fixed as suggested.